.github/workflows/docs-workflow.yml

```diff
@@ -5,18 +5,19 @@ on:
     types:
       - opened # a PR is created
       - synchronize # existing PR is updated
```

Comment:
Are you not going to remove these two?

Reply:
The synchronize event is still needed as it's what triggers a new preview if you make changes after adding a label (otherwise it's just a one time thing). You're right though, the opened event probably isn't needed anymore. Will remove that and add a comment to the synchronize event